### PR TITLE
Add wandb example in documentation

### PR DESCRIPTION
**Pull request by tamarl08**

### CODE REVIEW

File: doc/run_experiment.rst [hunk 1]
```diff
@@ -1686,22 +1686,28 @@ file using the :ref:`plot_learning_curves <plot_learning_curves>` utility script
...
       the confusion matrix as well as a table that shows per-label precision, recall and f-measure
       are logged for each job.
   *   **cross_validate**: Similar output logged as the `evaluate` task, with a separate job per CV fold.
   *   **learning_curve** The summary file is logged as a table, and all learning curve plots
       are logged as media artifacts.
+
+As an example, see `here <https://wandb.ai/etslabs/skll_titanic_example/reports/Titanic-Example-Report--Vmlldzo2ODAzNDY1?accessToken=jd3tssj6hs8rniby0rqgeahsm9oexppjgkjj3gdiisag49a0tkx1wnak806k4xjd>`__
+a report created on Weights & Biases, from the tutorial :ref:`titanic example <titanic_example>`.
+The report contains three sections, one per SKLL task, with a subset of the output tables and metrics that were logged during for the task.
+To view the full output and create your own reports, turn on :ref:`logging to Weights and Biases<wandb_credentials>` in the configuration ``Output`` section.
```
Comment (desilinguist) [inline doc/run_experiment.rst:1709]:
```suggestion
The above information logged to Weights & Biases can then be used to create informative reports for your
SKLL experiments. As an example, `here <https://wandb.ai/etslabs/skll_titanic_example/reports/Titanic-Example-Report--Vmlldzo2ODAzNDY1?accessToken=jd3tssj6hs8rniby0rqgeahsm9oexppjgkjj3gdiisag49a0tkx1wnak806k4xjd>`__ is
a report created on Weights & Biases, based on the data logged while running the tutorial :ref:`titanic example <titanic_example>`. The report contains three sections, one per SKLL task, with a subset of the output tables and metrics that were logged for each of the tasks. 

To view the full output and create your own reports, turn on :ref:`logging to Weights and Biases<wandb_credentials>` in the configuration ``Output`` section.
```

File: doc/run_experiment.rst [hunk 2]
```diff
@@ -1686,22 +1686,28 @@ file using the :ref:`plot_learning_curves <plot_learning_curves>` utility script
 
 Integration with Weights & Biases
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (desilinguist) [inline doc/run_experiment.rst:1688]:
```suggestion
--------------------------------------------
```

I think this should be one level up.

**Comment by desilinguist** [comment]
It'll also be nice to link  in the order direction – from where we define `wandb_credentials`  to this new top-level section.

**Comment by desilinguist** [comment]
Oh, BTW, the W&B report looks really nice! Thanks for dealing with their interface and docs that you don't like :)

**Review by desilinguist — APPROVED** [review]
LGTM! 🎉

**Review by damien2012eng — APPROVED** [review]

**Pull request merged**